packages/common/src/ui/components/popover/ClickPopover/ClickPopover.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { MouseEventHandler } from 'react';
+import { CloseIcon } from '@common/icons';
+import { Box, IconButton } from '@mui/material';
+import { BasePopoverProps } from '../BasePopover';
+
+export interface ClickPopoverProps {
+  Content: React.ReactElement<any, any>;
+  placement?: 'top' | 'bottom' | 'left' | 'right';
+  Popover: React.FC<Partial<BasePopoverProps>>;
+  show: MouseEventHandler<HTMLDivElement | HTMLButtonElement>;
+  hide: () => void;
+}
+
+export const ClickPopover: React.FC<ClickPopoverProps> = ({
+  children,
+  Content,
+  placement = 'left',
+  show,
+  hide,
+  Popover,
+}) => (
+  <>
+    <div style={{ cursor: 'pointer' }} onClick={show}>
+      {children}
+    </div>
+    <Popover placement={placement}>
+      <Box>
+        <IconButton
```

Comment:
ok, so maybe worth it. added a close button to the click popover, which isn't required on the hover version.

Reply:
Ah yes.

I like the close button but I also wonder if clicking away should close also.. that was my expectation.. or, at least when navigating away, I suppose.

I had a play and adding `<ClickAwayListener onClickAway={hide}>` around the `Box` seems to work fine if you think so also

Reply:
yeah - was undecided, my initial reaction was that click away should work. thanks for adding it in